src/WaterPokemon.java

```diff
@@ -0,0 +1,49 @@
+package src;
+
+public class WaterPokemon extends Pokémon{
+
+    //    2 private variables
+    private double height;
+    private double weight;
+
+    // 1 constructor
+    public WaterPokemon(String name, int hp, int xp, double height, double weight) {
+        super(name, hp, xp);
+        this.height = height;
+        this.weight = weight;
+    }
+
+    //    2 methods
+    public void hydroPump(double waterWeightToInflate) {
+        setWeight(getWeight() + waterWeightToInflate);
+    }
+
+    public void surf(double waterHeightToInflate) {
+        setHeight(getHeight() + waterHeightToInflate);
+    }
+
+    //    1 override
+    @Override
+    public void defence(int amount) {
+        super.defence((int) (amount*1.1));
+    }
```

Comment:
prima gebruik gemaakt van @Override

Reply:
Ik heb alleen een klein vermoeden dat je een denk fout hebt gemaakt hier, maar ik kan er naast zitten. Als je een water pokemon sterker wilt maken in defence moet je (amount  *  < 1) doen doen dus bijv. (amount * 0,9)  als je dan een hit van 10 krijgt is het (10 * 0,9) en verlies je maar 9 hp ipv 10.

Tenzij je een water pokemon slapper wilt maken in defence klopt het wat je gedaan hebt en heb ik niks gezegd